tests/sentry/integrations/vercel/test_integration.py

```diff
@@ -312,3 +312,29 @@ def test_upgrade_org_config_no_dsn(self):
         dsn.update(id=dsn.id, status=ProjectKeyStatus.INACTIVE)
         with self.assertRaises(IntegrationError):
             installation.update_organization_config(data)
+
+    @responses.activate
+    def test_ui_hook_options(self):
+        uihook_url = "/extensions/vercel/ui-hook/"
+        resp = self.client.options(path=uihook_url)
+        assert resp.status_code == 200
+
+    @responses.activate
+    def test_ui_hook_post(self):
+        uihook_url = "/extensions/vercel/ui-hook/"
+        with self.tasks():
+            self.assert_setup_flow()
+        integration = Integration.objects.get(provider=self.provider.key)
+        integration.update(
+            external_id="hIwec0PQ34UDEma7XmhCRQ3x",
+            metadata={
+                "configurations": {
+                    "icfg_Gdv8qI5s0h3T3xeLZvifuhCb": {"organization_id": self.organization.id}
+                }
+            },
+        )
+
+        data = b"""{"configurationId":"icfg_Gdv8qI5s0h3T3xeLZvifuhCb", "teamId":{}, "user":{"id":"hIwec0PQ34UDEma7XmhCRQ3x"}}"""
+
+        resp = self.client.post(path=uihook_url, data=data, content_type="application/json")
+        assert resp.status_code == 200
```

Comment:
@ceorourke This test should check to see if the correct URL to the integration is included in the response.